components/src/core/components/Input/RadioPills/RadioPill.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="oxd-radio-pill-wrapper">
+    <label :class="labelClasses" :style="style">
+      {{ $vt(label) }}
+      <input
+        type="radio"
+        @focus="onFocus"
+        @blur="onBlur"
+        @change="onChange"
+        v-bind="$attrs"
+        v-model="checked"
+        :value="value"
+        :disabled="disabled"
+        :readonly="readonly"
+        :class="classes"
+      />
+    </label>
+  </div>
+</template>
+
+<script lang="ts">
+import {computed, defineComponent, reactive, toRefs} from 'vue';
+import translateMixin from '@orangehrm/oxd/mixins/translate';
+
+export interface State {
+  focused: boolean;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  checkedProxy: any;
+}
+
+export default defineComponent({
+  name: 'oxd-radio-pill',
+  inheritAttrs: false,
+  props: {
+    name: {
```

Comment:
Will the ID attribute be added automatically to this widget when using with schema form?

Reply:
Yes it's being added as form_name + '_' + field_name to RadioPillGroup and from the group it assigns and id to the RadioPill